pandas/core/indexes/multi.py

```diff
@@ -559,6 +559,7 @@ def from_tuples(
             if isinstance(tuples, Index):
                 tuples = tuples._values
 
+            tuples = cast(np.ndarray, tuples)
```

Comment:
can you add a comment why we can do this for future readers. or a TODO if easy to explain how to rectify.